### DISCUSSION TITLE
How to hide absolute path

### DISCUSSION THREAD

**Discussion by Itsoon** [nvim-tree/nvim-tree.lua#2742] [Q&A] [answered]
How to hide/disable `~/.config/nvim/..`?
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/91259186/ea6dcff4-d99c-4780-9b4e-1daebaca1110)

**Comment by gegoune** [comment, marked as answer]
`:h nvim-tree.renderer.root_folder_label`

**Comment by Itsoon** [comment]
```lua
renderer = {
  root_folder_label = false,
}
```
